libs/build_api_v2.py

```diff
@@ -35,17 +39,52 @@
 USA_VACCINATION_START_DATE = datetime(2020, 12, 14)
 
 
-def _build_actuals(actual_data: dict) -> Actuals:
+_logger = structlog.get_logger()
+
+
+def _filter_demographic_field_and_remove_prefix(demographic_field: str, data: Dict):
+    return {
+        key.replace(demographic_field + ":", ""): value
+        for key, value in data.items()
+        if key.startswith(demographic_field)
```

Comment:
Oh, is this where the "all" bucket is skipped instead of getting output with the other distributions?